.github/workflows/s390x.yml

```diff
@@ -0,0 +1,37 @@
+name: s390x
+on:
+  push:
+    branches:
+      - master
+      - bytestring-0.11
+  pull_request: {}
```

Comment:
Same question. Even 28 minutes doesn't seem too bad – review will almost always be slower! ;)

Reply:
```suggestion
  pull_request: {} # Validate all PRs
```